apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.html.ts

```diff
@@ -44,159 +44,215 @@ export function InstructorAssessmentSettings({
+                      type="button"
+                      class="btn btn-sm btn-outline-secondary btn-copy"
+                      data-clipboard-text="${studentLink}"
+                      aria-label="Copy student link"
+                    >
+                      <i class="far fa-clipboard"></i>
+                    </button>
                     <button
-                      class="btn btn-sm btn-primary"
-                      href="#"
-                      data-toggle="modal"
-                      data-target="#deleteAssessmentModal"
+                      type="button"
+                      title="Student Link QR Code"
+                      aria-label="Student Link QR Code"
+                      class="btn btn-sm btn-outline-secondary js-qrcode-button"
+                      data-qr-code-content="${studentLink}"
                     >
-                      <i class="fa fa-times" aria-hidden="true"></i> Delete this assessment
+                      <i class="fas fa-qrcode"></i>
                     </button>
-                    ${Modal({
-                      id: 'deleteAssessmentModal',
-                      title: 'Delete assessment',
-                      body: html`
-                        <p>
-                          Are you sure you want to delete the assessment
-                          <strong>${resLocals.assessment.tid}</strong>?
-                        </p>
-                      `,
-                      footer: html`
-                        <input type="hidden" name="__action" value="delete_assessment" />
+                  </div>
+                </span>
+                <small class="form-text text-muted">
+                  The link that students will use to access this assessment.
+                </small>
+              </div>
+              ${
+                resLocals.authz_data.has_course_permission_view
+                  ? resLocals.authz_data.has_course_permission_edit &&
+                    !resLocals.course.example_course
+                    ? html`
+                        <a
+                          data-testid="edit-assessment-configuration-link"
+                          href="${resLocals.urlPrefix}/assessment/${resLocals.assessment
+                            .id}/file_edit/${infoAssessmentPath}"
+                        >
+                          Edit assessment configuration
+                        </a>
+                        in <code>infoAssessment.json</code>
+                      `
+                    : html`
+                        <a
+                          href="${resLocals.urlPrefix}/assessment/${resLocals.assessment
+                            .id}/file_view/${infoAssessmentPath}"
+                        >
+                          View assessment configuration
+                        </a>
+                        in <code>infoAssessment.json</code>
+                      `
+                  : ''
+              }
+            </div>
+            </form>
```

Comment:
You can tell that there's something invalid with this HTML because Prettier didn't format the indentation correctly here. In this case, the issue is that these closing tags are backwards:

```suggestion
            </form>
            </div>
```

You'll have to let Prettier do its thing after applying this suggestion.